src/main/webapp/i18n/de/student-question.json

```diff
@@ -3,7 +3,19 @@
         "studentQuestion": {
             "created": "Frage erfolgreich erstellt",
             "updated": "Frage erfolgreich aktualisiert",
-            "deleted": "Frage erfolgreich gelöscht"
+            "deleted": "Frage erfolgreich gelöscht",
+            "text": "Frage",
+            "creationDate": "gestellt am",
```

Comment:
The English version of this starts upper case, so this should too for consistency.